pyvista/plotting/tools.py

```diff
@@ -471,10 +516,10 @@ def parse_color(color, opacity=None, default_color=None):
             color = default_color
     if isinstance(color, str):
         color = string_to_rgb(color)
-    elif len(color) == 3:
-        pass
-    elif len(color) == 4:
-        color = color[:3]
+    elif isinstance(color, (list, tuple)) and len(color) in (3, 4) and np.all([isinstance(c, (int, float)) and 0 <= c <= 1 for c in color]):
```

Comment:
1. This line is way too long.
2. I'd go for first an `isinstance(color, (collections.abc.Sequence, np.ndarray)` test, then inside call `np.asarray`, then checking the dtype and values for the last check. Combination of `np.issubdtype` and `((0 <= c) & (c <= 1)).all()`.

I think the current implementation breaks twice for numpy arrays: both `isinstance` checks.

Reply:
This is marked as resolved but I don't think this was fixed yet (or at least the fix hasn't been pushed).

Reply:
Yea sorry, I was resolving them as I was fixing them locally to keep an overview of which things I still had to address. With the latest push everything should have been addressed :)